script.js

```diff
@@ -27,9 +27,11 @@ function startAnimation() {
         iterations: Infinity
     });
     pumpkin.animate([
-        {transform: 'translateY(0px)'},
```

Comment:
I can only comment on this so here I am, for some reason the animations start when you hover over the word start and when you leve the cofin.